bloomfilter/src/Data/BloomFilter/Hash.hs

```diff
@@ -173,6 +177,38 @@ instance Hashes CheapHashes where
     evalHashes = evalCheapHashes
     {-# INLINE evalHashes #-}
 
+instance Prim (CheapHashes a) where
+    sizeOfType# _ = 128#
+    alignmentOfType# _ = 64#
```

Comment:
These are in number of bytes

```suggestion
    sizeOfType# _ = 16#
    alignmentOfType# _ = 8#
```

I didn't immediately spot this until I ran `primLaws` tests from `quickcheck-classes`. Feel free to include this in the tests if you think it's useful:

```haskell

-------------------------------------------------------------------------------
-- Prim
-------------------------------------------------------------------------------

instance Arbitrary (CheapHashes a) where
  arbitrary = CheapHashes <$> arbitrary <*> arbitrary
  shrink (CheapHashes x y) = uncurry CheapHashes <$> shrink (x, y)

deriving instance Eq (CheapHashes a)

prop_primLaws :: Property
prop_primLaws = ioProperty $
    lawsCheck $ primLaws (Proxy @(CheapHashes Void))

testLawsCheck :: Laws -> TestTree
testLawsCheck (Laws className properties) =
    testGroup className [
        testProperty name prop
      | (name, prop) <- properties
      ]
```